zhaquirks/tuya/ts0601_electric_heating.py

```diff
@@ -135,14 +135,16 @@ class MoesBHTUserInterface(TuyaUserInterfaceCluster):
 
 class MoesBHT(TuyaThermostat):
     """Moes BHT-002GCLZB Thermostatic radiator valve."""
-
+    """BHT-003GBLZB Electric floor heating thermostat"""
```

Comment:
It doesn't like these comments in two lines. Put the references together. 

Maybe something like?:
`"""Tuya thermostat for devices like the Moes BHT-002GCLZB valve and BHT-003GBLZB Electric floor heating."""`